packages/web/src/pages/invalidEmail.tsx

```diff
@@ -0,0 +1,30 @@
+import { NextPage } from 'next';
+import { Button, Heading, HStack, Text, VStack } from '@chakra-ui/react';
+import { MarketingLayout } from '../components/Layout';
+
+const InvalidEmail: NextPage = () => {
+  return (
+    <MarketingLayout>
+      <VStack alignItems="center" paddingTop={8} w="100%" maxW="1200px" marginX="auto">
+        <Heading size="lg" textAlign="center" lineHeight="1.4">
+          Error: Make sure you have a verified &quot;.edu&quot; email linked with your GitHub
+          account!
+        </Heading>
+        <Text textAlign="center" maxWidth="60ch" fontSize="2xl" lineHeight="2">
```

Comment:
`lineHeight` shouldn't have effect here if you're setting the `fontSize`. Is there a reason it was included?